File: src/backend/TrafficCourts/Messaging/MessageContracts/Plea.cs
namespace TrafficCourts.Messaging.MessageContracts
{
    /// <summary>
    /// An enumeration of Plea Type on a DisputedCount record.
    /// </summary>
    public enum Plea
    {
        /// <summary>
        /// Unknown type (undefined). Must be index 0.
        /// </summary>
        Unknown,

        /// <summary>
        /// If the dispuant is pleads guilty, plea will always be Guilty. The dispuant has choice to attend court or not.
        /// </summary>
        G,

        /// <summary>
        /// If the dispuant is pleads not guilty, the dispuant will have to attend court.
        /// </summary>
        N
    }
}

File: src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/Plea.java
package ca.bc.gov.open.jag.tco.oracledataapi.model;

/**
 * An enumeration of Plea Type on a TicketCount record.
 */
public enum Plea implements ShortNamedEnum {

	/** Unknown type (undefined). Must be index 0. */
	UNKNOWN("U"),
	G("G"),
	N("N");

	private String shortName;

	private Plea(String shortName) {
		this.shortName = shortName;
	}

	@Override
	public String getShortName() {
		return shortName;
	}
}